server/index.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express');
+const morgan = require('morgan');
+const { review } = require('../db/models.js');
+
+const port = 3003;
+const app = express();
+app.listen(port, () => console.log(`listening on port ${port}`));
+
+app.use(express.static(`${__dirname}/../public`));
+app.use(morgan('dev'));
+
+app.get('/reviews', (req, res) => {
```

Comment:
May be worth setting up the route either this or next PR to receive a variable ID, e.g. `/reviews/:id`